assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js

```diff
@@ -47,7 +47,9 @@ const MOBILE_TO_DESKOP_MEDIA_QUERY =
 	'(min-width: 451px) and (max-width: 1280px';
 const X_LARGE_AND_ABOVE_MEDIA_QUERY = '(min-width: 1281px)';
 
-export default function UserCountGraph( { loaded, error, report } ) {
+export default function UserCountGraph( props ) {
+	const { loaded, error, report, gatheringData } = props;
```

Comment:
@eugene-manuilov Does that change make any difference? 🤔 Or we could have just added `gatheringData` to the list of destructured props in the function parameter?

Reply:
Gotcha! Thanks @eugene-manuilov